lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -214,8 +215,13 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
   }
 
   SILLocation::DebugLoc decodeDebugLoc(SILLocation Loc) {
-    if (Loc.isDebugInfoLoc())
-      return Loc.getDebugInfoLoc();
+    if (Loc.isDebugInfoLoc()) {
+      auto DLoc = Loc.getDebugInfoLoc();
+      if (Opts.DebugInfoFormat == IRGenDebugInfoFormat::CodeView)
+        // WinDbg does not handle column information.
```

Comment:
I think it would be helpful to add a more elaborate explanation here, explaining why it's necessary to strip the column information.